File: EmployeeService.Core/Entities/Employee.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace EmployeeService.Core.Entities
{
    public class Employee
    {
        [Key]
        public int Id { get; set; }
        public string? Name { get; set; }
        public string? Email { get; set; }

        [ForeignKey("CityId")]
        public int? CityId { get; set; }

    }
}

File: EmployeeService/ClientApp/emp-client-app/src/app/models/employee.ts
export class Employee {
  id: number;
  name: string;
  email: string;
  cityId: number;

  constructor(id: number,
    name: string,
    email: string,
    cityId: number,
  ) {
    this.id = id;
    this.name = name;
    this.email = email;
    this.cityId = cityId;
  }
}
